tendrl/ceph_integration/tests/test_config.py

```diff
@@ -0,0 +1,27 @@
+from mock import MagicMock
+import pytest
+from tendrl.ceph_integration import config
+
+
+class TestTendrlconfig(object):
+    def setup_method(self, method):
+        config.os.path.exists = MagicMock(return_value=True)
+
+    def test_with_default_path(self):
+        config.TendrlConfig()
+        config.os.path.exists.assert_called_with(
+            "/etc/tendrl/tendrl.conf"
+            )
+
+    def test_with_config_path(self, monkeypatch):
+        monkeypatch.setattr(config.os, 'environ', {"TENDRL_CONFIG": "/temp/"})
```

Comment:
pytest has a monkeypatch [function](http://docs.pytest.org/en/latest/monkeypatch.html#_pytest.monkeypatch.MonkeyPatch.setenv) specifically for patching environment variables: `setenv()`. Can that be used here?